MQTT v5 Feedbacks

Use this issue to report your feedback about MQTTv5 version. Positive feedbacks are welcome too so we know everything is working as expected 😄

Hi, I get the following error after updating to v5:

```
node_modules/mqtt/build/src/lib/client.d.ts:5:31 - error TS7016: Could not find a declaration file for module 'readable-stream'. '/Users/VWRTVRP/work/github/sdc-production/evo-viu-connector/node_modules/mqtt/node_modules/readable-stream/lib/ours/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/readable-stream` if it exists or add a new declaration (.d.ts) file containing `declare module 'readable-stream';`

5 import { DuplexOptions } from 'readable-stream';
                                ~~~~~~~~~~~~~~~~~

node_modules/mqtt/build/src/lib/store.d.ts:1:26 - error TS7016: Could not find a declaration file for module 'readable-stream'. '/Users/VWRTVRP/work/github/sdc-production/evo-viu-connector/node_modules/mqtt/node_modules/readable-stream/lib/ours/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/readable-stream` if it exists or add a new declaration (.d.ts) file containing `declare module 'readable-stream';`

1 import { Readable } from 'readable-stream';
```

--- a/package.json
+++ b/package.json
@@ -38,6 +38,7 @@
     ".": "./build/mqtt.js",
     "./package.json": "./package.json",
     "./*.map": "./build/*.js.map",
+    "./dist/*": "./dist/*.js",
     "./*": "./build/*.js"
   },
   "types": "build/mqtt.d.ts",

